src/Microsoft.Diagnostics.Runtime/src/DataReaders/Windows/WindowsProcessDataReader.cs

```diff
@@ -97,6 +97,18 @@ private void Dispose(bool _)
                 if (_process != IntPtr.Zero)
                     WindowsFunctions.NativeMethods.CloseHandle(_process);
 
+                if (_originalPid != 0)
+                {
+                    try
+                    {
+                        // Kill the cloned process as we don't need it anymore
+                        Process.GetProcessById(ProcessId).Kill();
+                    }
+                    catch (Win32Exception)
```

Comment:
This should be a blanket `catch {}` statement.

Usually I don't like adding those, but the original reason that block was removed is that exceptions were escaping this exact `catch (Win32Exception)` block and making it impossible to properly have a `using DataTarget dt = ...` statement without wrapping the whole thing in a try/catch.

Reply:
Or better yet, something like this (or similar):

``` C#
catch (Exception e)
{
    Trace.WriteLine($"Unable to kill the target process: {e.ToString()}");
}
```

Reply:
Done!